src/token/Rift.sol

```diff
@@ -1,58 +1,94 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
-pragma solidity 0.8.11;
+pragma solidity ^0.8.11;
```

Comment:
we did chat about this but i think for prod we should keep this consistent with the rest of the repo, without the `^`. can potentially use asdf to manage solc versions (https://github.com/refillic/asdf-solidity) also happy to walk through it tm